.changeset/eleven-balloons-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+---
+
+Add visibility filter on GithubEntityProvider
```

Comment:
suggestion: It would be great to include a code snippet or reference link  to the docs showcasing the new configuration.

Reply:
similarly to this https://github.com/backstage/backstage/commit/8749df3d028ceebb22cfec5cf31606ca970c7284 ?

Reply:
Yes, something like this would work.

Reply:
added a link